src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/patch/SpoofSignatureVerificationPatch.kt

```diff
@@ -82,6 +84,25 @@ class SpoofSignatureVerificationPatch : BytecodePatch(
 
         } ?: return OpenCronetDataSourceFingerprint.toErrorResult()
 
+        // hook override subtitles
+        SubtitleWindowSettingsConstructorFingerprint.result?.let {
+            it.mutableMethod.apply {
+                addInstructions(
+                    0,
+                    """
+                        invoke-static {p1, p2, p3, p4, p5}, $INTEGRATIONS_CLASS_DESCRIPTOR->getSubtitleWindowSettingsOverride(IIIZZ)[I
+                        move-result-object v0
+                        const/4 v1, 0x0
+                        aget p1, v0, v1
```

Comment:
Can you add comments to what those are and why they are required for future reference?